handbook/engineering/code-insights/index.md

```diff
@@ -0,0 +1,61 @@
+# Code insights team
+
+The code insights team will be responsible for building and delivering [**code insights**](https://docs.google.com/document/d/1EHzor6I1GhVVIpl70mH-c10b1tNEl_p1xRMJ9qHQfoc/edit) to engineering leaders, empowering data-driven decisions in engineering organizations.
+
+<img src="./screenshot.svg" alt="Screenshot of a code insights dashboard with graphs" />
+
+## What is code insights?
+
+Code insights is the first feature in Sourcegraph that can tell you things about your code at a **high level**. Here's a [demo](https://www.youtube.com/watch?v=XqeRb6Mc4Co) of a code insights prototype.
+
+Sourcegraph is in the unique position to give these insights because we have universal code search: To know _anything_ about your code at a high level means you must know _everything_ about your code at a low level.
+
+Code insights connects a lot of features that Sourcegraph already has and builds on top of them.
+We go beyond single-step code intelligence and search to connect the full cycle of analyzing (code intelligence), monitoring (code insights), and actionably changing a codebase (campaigns).
+
+Code insights will also allow to combine these Sourcegraph-created metrics with other third-party data sources.
+
+Code insights is the first feature primarily built for non-search-based user personas.
+
+## Goals 
+
+**Problem:** Code insights solves two problems. 
+
+1. The Sourcegraph sales cycle is unusual because although we consistently wow our users, the economic decision-maker is usually not one of these users. Instead, the people with the power to sign a contract with us are higher up within an organization and usually depend on running a trial to fully understand the value of Sourcegraph. If code insights answers these personas' higher-level questions about codebases that our core features do not currently answer, it could dramatically speed up our sales cycle as well as our sales pipeline.
+
+1. Sourcegraph has lots of information about your code, but currently only exposes specific, low-level information at the file or directory level. Engineers and engineering managers/directors/VPs have lots of questions about their codebase that Sourcegraph can answer but currently does not. 
+
+**Milestones:**
+
+_Because code insights is an entirely new feature and closely informed by customer feedback, the further in the future goals get, the more flexible the order of these milestones is (especially when it comes to interleaving "business" milestones like beta/GA/paid with feature milestones)._ 
+
+1. 🔄 Code insights prototypes let us gather qualitative feedback to guide the initial product direction. 
+1. We can quantitatively measure the adoption of code insights prototypes. 
```

Comment:
```suggestion
1. We have decided on and implemented a metric to quantitatively measure the adoption of code insights prototypes. 
```

Reply:
(Made this change and linked the RFC as well.)